packages/next/build/webpack/plugins/profiling-plugin.ts

```diff
@@ -72,7 +53,7 @@ export class ProfilingPlugin {
       }
     })
     stopHook.tap(pluginName, () => {
-      stackPop(this.compiler, span, spanName)
+      stackPop(this.compiler, span)
```

Comment:
Are the attributes returned on line 66 for `webpack-compile` supposed to be nested under the `attributes` field? Looks like this isn't done elsewhere and the attributes are set directly e.g. instead of `{ attributes: { name: compiler.name } }` just `{ name: compiler.name }`?

Reply:
That's a good catch. Thank you!